Dockerfile

```diff
@@ -4,10 +4,10 @@ MAINTAINER SD Elements
 RUN apt-get update
 RUN apt-get install -y nodejs nodejs-legacy npm git libkrb5-dev
```

Comment:
@sibartlett `libkrb5-dev` is optional now, correct, if you aren't using the kerb/ldap plugins.

Unrelated to this change.